elliott/elliottlib/cli/find_bugs_sweep_cli.py

```diff
@@ -261,16 +270,14 @@ def get_assembly_bug_ids(runtime, bug_tracker_type):
 
 
 def categorize_bugs_by_type(bugs: List[Bug], advisory_id_map: Dict[str, int],
-                            permitted_bug_ids, permissive=False, major_version: int = 4):
+                            permitted_bug_ids, operator_advisory, permissive=False, major_version: int = 4):
```

Comment:
I'm thinking we don't explicitly need to know the concept of an advance release here.
If you see the code, we're fetching advisories from advisory_id_map and then wherever the build is attached we assign the bug there. So we can do something like

```python3
bugs_by_type: Dict[str, type_bug_set] = {kind: set() for kind in advisory_id_map.keys()}
```